genericjpa/copyright.txt

```diff
@@ -0,0 +1,4 @@
+Martin Braun
```

Comment:
this is also redundant, the copyright is applied from the root of the repository. Your name is there already right?